react/src/components/atoms/forms/TextArea/index.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './style.css';
+
+class TextArea extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { value: '' };
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  static getDerivedStateFromProps(nextProps, prevState) {
+    if (nextProps.defaultText !== prevState.value) {
+      return{
+        value: nextProps.defaultText
+      };
+    }
+    return null;
+  }
```

Comment:
We have this pattern in a lot of our components, but I wonder if we need it.
Outside of Storybook/Knobs, would this component ever need to reset its state because of a prop change?

**Anti-pattern: Erasing state when props change**
https://reactjs.org/blog/2018/06/07/you-probably-dont-need-derived-state.html#anti-pattern-erasing-state-when-props-change